common/internal_model/src/main/scala/weco/catalogue/internal_model/work/InstantRange.scala

```diff
@@ -39,6 +39,30 @@ object InstantRange {
       label
     )
 
+  val positiveInfinity = LocalDate
+    .of(9999, 12, 31)
+    .atStartOfDay()
+    .plusDays(1)
+    .minusNanos(1)
+    .toInstant(ZoneOffset.UTC)
+
+  val negativeInfinity = LocalDate
+    .of(-9999, 1, 1)
+    .atStartOfDay()
+    .toInstant(ZoneOffset.UTC)
```

Comment:
Minor: possibly worth a comment that Sierra uses 9999, and we didn't just pick it arbitrarily?

Also make these values private unless you're using them elsewhere?